index.js

```diff
@@ -397,7 +397,9 @@ class Encore {
      *      const nodeExternals = require('webpack-node-externals');
      *
      *      Encore.addExternals(
-     *          nodeExternals()
+     *          // add any valid externals you have
+     *          nodeExternals(),
+     *          /^(jquery|\$)$/i
```

Comment:
That PR is the one that doesn't use rest parameters (as opposed to #472), so you'd need to put these two lines into an array.